.rubocop.yml

```diff
@@ -139,11 +136,75 @@ Style/DateTime:
     - "rakelib/evss.rake"
     - "lib/evss/auth_headers.rb" #DateTime is needed for date formatting here
 
-
 # Skipping for now, should revisit:
 Rails/HasManyOrHasOneDependent:
   Exclude:
     - "app/models/education_benefits_claim.rb"
     - "app/models/saved_claim/education_benefits.rb"
     - "app/models/saved_claim.rb"
     - "app/models/terms_and_conditions.rb"
+
+# Using .empty? with the safe navigation operator seems to be the correct call in this situation
+Lint/SafeNavigationWithEmpty:
+  Exclude:
+    - 'lib/hca/configuration.rb'
```

Comment:
Let's get rid of this exclusion by changing `lib/hca/configuration.rb` to use `present?` instead of `empty?`
```
    CERT_STORE = if Settings.hca.ca.present?
                   cert_store(Settings.hca.ca)
                 else
                   nil
                 end
```

Reply:
```ruby
Settings.hca.ca.present? ? cert_store(Settings.hca.ca) : nil
```